code/controllers/subsystem/economy.dm

```diff
@@ -29,12 +41,27 @@ SUBSYSTEM_DEF(economy)
 	/// Mail Holiday: AKA does mail arrive today? Always blocked on Sundays, but not on bee, the mail is 24/7.
 	var/mail_blocked = FALSE
 
+
+	/// checking `if(HAS_TRAIT(SSstation, STATION_TRAIT_UNITED_BUDGET))` costs (a little bit) more resource. this is faster.
+	var/united_budget_trait = FALSE
```

Comment:
yeah no don't do this please

cache it in the proc that uses it, but not on a global scale